src/twisted/mail/relaymanager.py

```diff
@@ -578,15 +582,15 @@ def notifyFailure(self, relay, message):
         @param message: The path of the file holding the message.
         """
         if self.noisy:
-            log.msg("could not relay " + message)
+            log.msg(b"could not relay " + message)
```

Comment:
I think it would be better to convert `message` to unicode  (e.g. by `repr` ?) than to emit the log text as a bytestring.